routing/router.go

```diff
@@ -2331,3 +2332,10 @@ func (r *ChannelRouter) IsStaleEdgePolicy(chanID lnwire.ShortChannelID,
 func (r *ChannelRouter) MarkEdgeLive(chanID lnwire.ShortChannelID) error {
 	return r.cfg.Graph.MarkEdgeLive(chanID.ToUint64())
 }
+
+// edge is a combination of a channel and the node pubkeys of both of its
+// endpoints.
+type edge struct {
```

Comment:
define above first use?

Also, `edge` is possibly a confusing name since it used for all sorts of things. Do we need to introduce yet another edge/channel definition?

Reply:
Moved up.

Yes it is another variation indeed, but it only exists temporary until we move to passing the full route into mission control in #3077.